samcli/lib/providers/sam_stack_provider.py

```diff
@@ -107,11 +110,21 @@ def _extract_stacks(self) -> Dict[str, Stack]:
             stack: Optional[Stack] = None
             if resource_type == SamLocalStackProvider.SERVERLESS_APPLICATION:
                 stack = SamLocalStackProvider._convert_sam_application_resource(
-                    self._template_directory, self._stack_path, name, resource_properties
+                    self._template_directory,
+                    self._stack_path,
+                    name,
+                    resource_properties,
+                    self._global_parameter_overrides,
+                    self._ignore_code_extraction_warnings,
                 )
             if resource_type == SamLocalStackProvider.CLOUDFORMATION_STACK:
                 stack = SamLocalStackProvider._convert_cfn_stack_resource(
-                    self._template_directory, self._stack_path, name, resource_properties
+                    self._template_directory,
+                    self._stack_path,
+                    name,
+                    resource_properties,
+                    self._global_parameter_overrides,
```

Comment:
this was previously `None` for `global_parameter_overrides`. It should have been `self._global_parameter_overrides`. I added it just for convenience. 

The `global_parameter_overrides` does not make a difference in our code because all `Stack` objects that are actually used by other classes are created in `get_stacks()`. I will simplify this class in the future. For now, it does not matter whether `global_parameter_overrides` is provided or not.